src/api/wallet.js

```diff
@@ -53,6 +53,54 @@ const walletApi = {
     });
   },
 
+  /**
+   * Call get token info API
+   *
+   * @param {string} uid Token uid to get the info
+   * @param {function} resolve Method to be called after response arrives
+   *
+   * @return {Promise}
+   * @memberof ApiWallet
+   * @inner
+   */
+  getTokenInfo(uid, resolve) {
```

Comment:
I think we are using "tokenInfo" for two different things. It is both the information of the token inside the create token transaction and the general token info from the API, right? Maybe we should rename one of them.

Reply:
I agree. I've renamed this API to to generalTokenInfo, what do you think?

Reply:
Seems good. Thanks!